src/emcpy/plots/create_map.py

```diff
@@ -0,0 +1,297 @@
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from emcpy.plots.map_tools import Domain, MapProjection
+
+
+__all__ = ['CreateMap']
+
+
+class CreateMap:
+    """
+    Creates a map axes on a figure to plot data.
+    """
+
+    def __init__(self, fig=plt.figure(figsize=(12, 8)),
+                 domain=Domain('global'),
+                 projection=MapProjection('plcarr')):
+        """
+        CreateMap constructor
+
+        Args:
+            fig : (matplotlib.figure) Figure object to plot on
+            domain : (object) domain from emcpy.plots.map_tools.Domain()
+            projection : (object) projection from emcpy
+                         emcpy.plots.map_tools.MapProjection()
+        """
+
+        self.fig = fig
+        self.domain = domain
+        self.projection = projection
+        ax = fig.add_subplot(1, 1, 1, projection=projection.projection)
+
+        if projection.__str__() not in ['npstere', 'spstere']:
```

Comment:
The `magic` method `__str__` is defined in the `MapProjection` class.  
Use it `pythonic`-ally.
```suggestion
        if str(projection) not in ['npstere', 'spstere']:
```

Reply:
General rule of thumb, you are not supposed to call the `dunder` or `magic` methods explicitly.  They are reserved for classes internal representation.